### PR TITLE
 Oshinko-webui now uses console-style calls for all actions. 

### PR DESCRIPTION
**Pull request by crobby**
Should be merged after https://github.com/radanalyticsio/radanalyticsio.github.io/pull/115

### CODE REVIEW

**Comment by elmiko** [comment]
tested this against
```
$ oc version
oc v3.6.0+387485d-31
kubernetes v1.6.1+5115d708d7
features: Basic-Auth

Server https://shift.opb.studios:8443
openshift v3.6.0+c4dd4cf
kubernetes v1.6.1+5115d708d7
```

everything seemed to work fine and cluster deployed as normal.

only things i noticed that were odd:
1. the exposed route link is defaulting to the proxy endpoint (although i could find the proper route in the card)
2. the green background text behind the "Running" status text is gone

**Comment by crobby** [comment]
@elmiko  
The route link logic in the console seems to be governed by "most specifically defined route", which in this case is the proxy route...and is undesirable for us.  On my list is a way to game that system to get the behavior that we want.

The green is gone by design.  Green is not part of the console, so we no longer use it.

**Comment by elmiko** [comment]
thanks for the explanations, i don't think the route thing is a blocker but we will need an improvement at some point.

i'm good to merge this

File: scripts/launch.sh [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-
 # Parse the spark default image from the oshinko CLI version info
 # For a version of oshinko-web that doesn't use the CLI, the value would just be set here
 SPARK_DEFAULT=$($(dirname $0)/oshinko version | sed -rn 's/Default spark image:\s(.*)/\1/p')
```
Comment (tmckayus) [inline scripts/launch.sh]: This can't actually still work, can it?  The install of the CLI was removed.
We need a new mechanism to set the spark version.

**Pull request merged**